stdlib/3/_codecs.pyi

```diff
@@ -1,51 +1,61 @@
 """Stub file for the '_codecs' module."""
 
-from typing import Any, AnyStr, Callable, Tuple, Optional, Dict
+import sys
+from typing import Any, Callable, Tuple, Optional, Dict, Union
 
 import codecs
 
 # For convenience:
 _Handler = Callable[[Exception], Tuple[str, int]]
+_String = Union[bytes, str]
```

Comment:
I think I'd rather do that when I actually merge the stubs since I don't know for sure that Python 2 accepts the exact same types.